src/main/java/org/dcsa/ebl/controller/TransportDocumentController.java

```diff
@@ -29,6 +29,7 @@
       @PathVariable @Size(max = 20) String transportDocumentReference) {
     return transportDocumentService
         .findByTransportDocumentReference(transportDocumentReference)
+      .map(transportDocumentTO -> transportDocumentTO)
```

Comment:
Maybe a `.cast(...)` instead (or `.<Type>findByTransportDocumentReference(transportDocumentReference)`)

Reply:
This was used during debugging only to properly set a breakpoint. is does not serve a real purpose, so is deleted